ott/tools/gaussian_mixture/scale_tril.py

```diff
@@ -157,24 +157,40 @@ def _flatten_cov(cov: jnp.ndarray) -> jnp.ndarray:
     return (cost_fn.norm(x0) + cost_fn.norm(x1) +
             cost_fn.pairwise(x0, x1))[...,]
 
-  def transport(
-      self, dest_scale: 'ScaleTriL', points: jnp.ndarray
-  ) -> jnp.ndarray:
-    """Transport between 0-mean normal w/ current scale to one w/ dest_scale.
+  def transport_scale_matrix(self, dest_scale: 'ScaleTriL') -> jnp.ndarray:
+    """Scaling matrix used in transport between 0-mean Gaussians.
+
+    Sigma_mu^{-1/2} @
+      [Sigma_mu ^{1/2} Sigma_nu Sigma_mu ^{1/2}]^{1/2}
+    @ Sigma_mu ^{-1/2}
 
     Args:
       dest_scale: destination Scale
-      points: points to transport
 
     Returns:
-      Points transported to a Gaussian with the new scale.
+      Gaussian scaling matrix, same dimension as self.covaraince()
```

Comment:
self.covariance

Reply:
covariance was defined as a method i.e. without the property decorator, I can change in doctoring though